### PR TITLE
Tier 0.5 quests minor modifications

### PR DESCRIPTION
**Pull request by gabrielnoguerales**
- Modify dependancies of quests to reflect dependancy of Steel or Wrought Iron
 - Added some tips for the "Time to get some Steel" quest

### CODE REVIEW

File: config/betterquesting/DefaultQuests.json [hunk 1]
```diff
@@ -335210,9 +335230,9 @@
       "home_image:8": "dreamcraft:textures/gui/gtnh_title.png",
       "party_enable:1": 1,
       "pack_version:3": 2110,
-      "editMode:1": 0,
+      "editMode:1": 1,
```
Comment (Dream-Master) [inline config/betterquesting/DefaultQuests.json:335233]: please disable edit mode or set this back to 0

File: config/betterquesting/DefaultQuests.json [hunk 2]
```diff
@@ -4473,7 +4473,9 @@
     "47:10": {
       "questID:3": 47,
       "preRequisites:11": [
-        46
+        46,
+        76,
+        948
```
Comment (Dream-Master) [inline config/betterquesting/DefaultQuests.json:4478]: I am not ingame what quests are 76 and 948

Comment (Dream-Master) [reply config/betterquesting/DefaultQuests.json:4478]: or add more text in your pr text field. Thanks

File: config/betterquesting/DefaultQuests.json (hunk 1 above)
Comment (gabrielnoguerales) [reply config/betterquesting/DefaultQuests.json]: Done, thanks.

File: config/betterquesting/DefaultQuests.json (hunk 2 above)
Comment (bombcar) [reply config/betterquesting/DefaultQuests.json:4478]: 76 - Time to Get Some Steel
948 - Better Than Iron
46 - More Advanced Alloys

Comment (gabrielnoguerales) [reply config/betterquesting/DefaultQuests.json:4478]: Thanks @bombcar

File: config/betterquesting/DefaultQuests.json [hunk 3]
```diff
@@ -7666,7 +7683,7 @@
...
+          "desc:8": "You need steel? Uhh... fine. Let\u0027s get busy...\n\nHint: The bricked blast furnace can share walls with other ones, just like most GT multi blocks. How will this affect the air quality though...\n\n§3If you are short on Stone Dust, you can get more by macerating cobblestone.\n\nIf you spawn near a biome with Hardened Clay you can macerate it to get the clay dust you need. You can also use brick blocks from Villages to get brick dust in the macerator.\n\nIf you can, craft a WorkTable from Forestry it will make the crafting a lot easier as you can craft with all your buckets in one click\n\n§4You\u0027ll need 432 unfired clay brick, 432 brick, 108 clay and brick dust, 216 compressed fireclay, and 216 firebrick. Collect all the required materials for these first, or you\u0027ll have to do it again. Also, don\u0027t make the firebrick blocks until you collect all the gypsum you need unless you want to be forced to get more."
```
Comment (bombcar) [inline config/betterquesting/DefaultQuests.json:7686]: The comment text change is very good (though I'd word the sentence as:

Crafting the WorkTable from Forestry can make it easier as you can use all the buckets in one click.

**Review by bombcar — COMMENTED** [review]
As for the other changes, while well intentioned, I think they make the quest page very "spiderweby" - perhaps instead some could be entirely hidden until the appropriate time?

image

vs

image

**Comment by gabrielnoguerales** [comment]
@bombcar I agree it makes the quest page very "spiderweby", some quests need to be reorder to adjust to make them more clear about their dependancies. I can try to do it but it overwhelms me a little bit as I havent played before with betterquest.

@bombcar @Dream-Master I pushed more changes to the pr if you could take a look

File: config/betterquesting/DefaultQuests.json (hunk 3 above)
Comment (gabrielnoguerales) [reply config/betterquesting/DefaultQuests.json:7686]: Great

**Review by bombcar — APPROVED** [review]

**Comment by bombcar** [comment]
>  What exactly was the goal with some of the changes? Make it clear you need to do another quest? Perhaps the solution instead of dependencies across the map would be to add some quest text - "If you can make ..."

**Comment by gabrielnoguerales** [comment]
The purpose was to not have quests unlocked that you can't do before other ones. In other words so all unlocked quests are compleatable without needing to complete another one before. 
It is specially useful when deciding what quest to make as it deletes the path you currently can't take.

**Comment by bombcar** [comment]
> The purpose was to not have quests unlocked that you can't do before other ones. In other words so all unlocked quests are compleatable without needing to complete another one before.
> It is specially useful when deciding what quest to make as it deletes the path you currently can't take.

The problem with that is very quickly you get many quest that depend on each other - GTNH isn't really a "railroad" pack where you have to follow one particular quest line - you're meant to learn how to use NEI and realize what you need to build (later on referencing other entire tabs).

I do agree it can be confusing early in steam, which is why I think quest text updates may help - and prevent the spiderweb of cross-connected quests.

**Comment by gabrielnoguerales** [comment]
> > The purpose was to not have quests unlocked that you can't do before other ones. In other words so all unlocked quests are compleatable without needing to complete another one before.
> > It is specially useful when deciding what quest to make as it deletes the path you currently can't take.
> 
> The problem with that is very quickly you get many quest that depend on each other - GTNH isn't really a "railroad" pack where you have to follow one particular quest line - you're meant to learn how to use NEI and realize what you need to build (later on referencing other entire tabs).
> 
> I do agree it can be confusing early in steam, which is why I think quest text updates may help - and prevent the spiderweb of cross-connected quests.

I agree and its one of the things I am enjoying the most, being able to have multiple paths to take (Fe. I took a detour on Steam tier to do some bees). But when you arrive to Steam tier you unlock quests that you can only make after having materials that puts you near the end of the tier so I though it was better to have those quests unlocked when you have the materials needed to complete them.

**Comment by bombcar** [comment]
I believe it is possible to hide a quest until one is completed without making it a dependency (so no line) - but not exactly sure how to do that.

**Comment by Dream-Master** [comment]
@gabrielnoguerales sorry but this is a real mess. We cant let this like it is now. 
1. i make a reverse of the pr
2. you find a way to rearraged the icons no lines corssing
3. someone can add some code to BQ remove this lines (so you can choose if you want to show the dependencies or not)

**Comment by bombcar** [comment]
BetterQuesting now supports IMPLICIT which is what you’d want to use for these (the long lines would only appear on hover over)

**Comment by Dream-Master** [comment]
Are the implict works on single line or all lines are affected ?

**Comment by bombcar** [comment]
> Are the implict works on single line or all lines are affected ?

I need to check but it looks like each one can choose - default normal.

**Comment by bombcar** [comment]
let me try update - don't revert yet

**Comment by bombcar** [comment]
All fixed in https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/pull/8938 - requires latest BQ

**Pull request merged**